packages/auth/package.json

```diff
@@ -18,7 +18,8 @@
     "prepack": "pnpm build",
     "postinstall": "pnpm copy-op-schemas",
     "copy-op-schemas": "cp ./node_modules/@interledger/open-payments/dist/openapi/specs/auth-server.yaml ./node_modules/@interledger/open-payments/dist/openapi/specs/schemas.yaml ./src/openapi/specs/",
-    "copy-files": "cp src/graphql/schema.graphql dist/graphql/ && cp -r ./src/openapi ./dist/"
+    "copy-files": "cp src/graphql/schema.graphql dist/graphql/ && cp -r ./src/openapi ./dist/",
+    "dev": "nodemon --watch src --ext ts --exec \"tsc && pnpm copy-files && node --inspect=0.0.0.0:9229 dist/index.js\""
```

Comment:
This is run from the dev dockerfile. In plain english, it is:

Copying over the gql/openapi schema files, rebuilding, and starting the sever with the debugger. Then on the docker container's src file change, it is repeating the copy/rebuild/server start. The docker container's src file is a bind mount of the host machine's src file. Thus saving changes triggers the server restart.